src/social-network-provider/twitter.com/utils/delegate.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * @desc
+ *  this file contains utils that used to
+ *  delegate some tasks to foreground by background.
+ *  we call foreground tabs 'worker' here.
+ */
+import { anyHostUrl, anyHostUrlMobile } from './url'
+import { shuffle } from 'lodash-es'
+
+export const workerAllocate = async () => {
```

Comment:
I think the naming is confusing that the variables doesn't indicates what network does it serve. Like `anyHostUrl` is confusing in the global scope (auto importing). `anyHostURLForTwitter` will be better?

Reply:
renamed